coalib/results/Diff.py

```diff
@@ -70,6 +71,24 @@ def from_string_arrays(cls, file_array_1, file_array_2, rename=False):
         return result
 
     @classmethod
+    def from_unidiff(cls, diff_array):
+        """
+        Creates a Diff object from given unified diff.
+
+        :param diff_array: list of lines of unified diff.
```

Comment:
So for test cases, shall I leave them as arrays (and do join) or make them strings as well.